package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@hashicorp/react-components",
   "version": "0.0.0",
   "dependencies": {
-    "@hashicorp/mktg-global-styles": "^4.0.0",
+    "@hashicorp/mktg-global-styles": "^4.1.0",
```

Comment:
Pulls in new global grid class https://github.com/hashicorp/mktg-global-styles/releases/tag/4.1.0